src/log.py

```diff
@@ -0,0 +1,107 @@
+"""
+log
+ 
+By: Norma Chang
+"""
+import json
+import datetime
+import time
+ 
+ 
+class Log:
+    """
+    Log class.
+ 
+    Generates a log file that contains the data from the properties of an instance of the leader/follower class.
+ 
+    Array of dictionaries is generated when the log file is loaded.
+    """
+    def __init__(self):
+        self._start_time = str(datetime.datetime.now())
+
+
+    def log_to_file(self, robot_obj, role):
```

Comment:
this makes a lot more sense as `vehicle` instead of `robot_obj`

We don't refer to autonomous cars as robots and the leader is not a robot.

Reply:
changed name to vehicle

Reply:
Is this a transparency concern?
Would it be better if I created a dictionary in follower/leader then just pass the dictionary to the log instead of the whole object?